libs/react-helpers/src/lib/format/number.ts

```diff
@@ -55,19 +55,25 @@ export const formatNumber = (
 };
 
 export const addDecimalsFormatNumber = (
-  rawValue: string | number,
-  decimalPlaces: number,
+  rawValue?: string | number | null,
```

Comment:
question (blocking): I'm also uncomfortable with these changes. These are simple formatting functions. If you need to pass undefined here something is probably wrong with your types and/or code elsewhere

Reply:
updated.